Source/Core/DolphinWX/AboutDolphin.cpp

```diff
@@ -36,7 +36,8 @@ AboutDolphin::AboutDolphin(wxWindow* parent, wxWindowID id, const wxString& titl
       _("\n"
         "Dolphin is a free and open-source GameCube and Wii emulator.\n"
         "\n"
-        "This software should not be used to play games you do not legally own.\n");
+        "This software should not be used to play games you do not legally own.\n"
+        "\nWith contributions by: Connor Roth, Calvun Williams, Justin Futrell, Andrew Nguyen\n");
```

Comment:
There is already a reference to GitHub's contributors page in the dialogue, no need to add your names to the list :P